composer.json

```diff
@@ -136,7 +136,7 @@
         "guzzlehttp/promises": "^1.4",
         "league/html-to-markdown": "^5.0",
         "masterminds/html5": "^2.7.2",
-        "monolog/monolog": "^1.25.1|^2",
+        "monolog/monolog": "^3",
```

Comment:
we don't allow v3 on branch 6.4? that's surprising. can't we fix that?

Reply:
Indeed, I guess it's because people don't use symfony/symfony anymore.

Anyway, I'll open a PR on 6.4 to add some deprecation notice

Reply:
If we do that, the bridge won't be tested with Monolog 2 anymore, will it?

Currently:
* same-deps run: Monolog 2
* high-deps run: Monolog 3
* low-deps run: Monolog 1